data/json/furniture_and_terrain/furniture-recreation.json

```diff
@@ -638,5 +638,45 @@
         { "item": "scrap", "count": [ 15, 60 ] }
       ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_archery_target_box",
+    "name": "box archery target",
+    "description": "A soft target for archery consisting of tightly-packed cloth stuffed into a cardboard box, with a bullseye marked on the front.",
+    "symbol": "@",
+    "color": "brown",
+    "move_cost_mod": 9,
+    "required_str": 5,
+    "flags": [ "TRANSPARENT", "FLAMMABLE", "EASY_DECONSTRUCT" ],
+    "crafting_pseudo_item": "pseudo_archery_target_box",
+    "deconstruct": { "items": [ { "item": "archery_target_box", "count": 1 } ] },
+    "bash": {
+      "str_min": 1,
```

Comment:
If I were to manually shoot this as is implied in the practice recipe, it would break :I

Reply:
oh:( 
How would I fix that? raise str_min?

Reply:
> oh:( How would I fix that? raise str_min?

Yees :)

Reply:
done!